### ISSUE TITLE
Please double check your research findings

### ISSUE THREAD

**Issue by joachimmetz** [jklepsercyber/defender-detectionhistory-parser#7]
From [README](https://github.com/jklepsercyber/defender-detectionhistory-parser#readme)

> The creation of these files is an after-product of Windows Defender's real-time/cloud-delivered protection(RTP) blocking threats such as Potentially Unwanted Applications (PUAs), viruses, worms, trojans, etc. 

The files appear to be generated even with cloud-delivered protection turned off. 

> The file begins with a header, 0x0800000008, taking up the first 5 bytes in every known scenario

There are files under the MputHistory directory that start with the same 5 bytes that contain different information. So it does not look like to be a "signature" (as in something that uniquely identifies the DetectionHistory files)

**Comment by joachimmetz** [comment]
> The file begins with a header, 0x0800000008, taking up the first 5 bytes in every known scenario

The format looks like a simple value type container format where the first 5 bytes are part of the first value.

**Comment by jklepsercyber** [comment]
Thanks for the detailed proof-read, Joachim. Still new to the field so some of the terminology is still coming to me, regarding "value type container formats" 😄 will look into these though, and double check some previous points I made as well. I appreciate the tips!

**Comment by joachimmetz** [comment]
> Thanks for the detailed proof-read, Joachim

Note that this is anything far from an actual proof read. I mostly only skimmed the material, and my observations did not match yours.

> value type container formats

TL;DR a format that defines pairs of value types and data, e.g. type: string, value: "TEST" or type: integer, value: 99

>  Still new to the field so some of the terminology is still coming to me

Maybe interesting contextual read about the importance about reproducible test data https://osdfir.blogspot.com/2020/09/testing-digital-forensic-data.html

Not to influence you too much but documentation of my current findings: https://github.com/libyal/dtformats/blob/main/documentation/Windows%20Defender%20scan%20DetectionHistory%20file%20format.asciidoc
